appveyor.yml

```diff
@@ -10,6 +10,14 @@ environment:
 
   matrix:
     - TARGET_ARCH: x86
+      CONDA_PY: 27
+      CONDA_INSTALL_LOCN: C:\\Miniconda
+
+    - TARGET_ARCH: x64
+      CONDA_PY: 27
+      CONDA_INSTALL_LOCN: C:\\Miniconda-x64
+
+    - TARGET_ARCH: x86
```

Comment:
That is not really the goal of `conda-forge` these test put quite a strain on our 2nd most valuable resource, the CIs.